### PR TITLE
Adds getManagedInstances and deleteInstances to InstanceGroupManger API

### PR DESCRIPTION
**Pull request by modeswitch**
Partially addresses #103.

- [x] Tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

### CODE REVIEW

**Comment by modeswitch** [comment]
@JustinBeckwith One more for you!

File: src/instance-group-manager.js [hunk 1]
```diff
@@ -153,7 +155,154 @@ class InstanceGroupManager extends common.ServiceObject {
...
+   * }, callback);
+   *
+   * //-
+   * // If the callback is omitted, we'll return a Promise.
+   * //-
+   * instanceGroupManager.listManagedInstances().then(function(data) {
+   *   const vms = data[0];
+   * });
+   */
+  listManagedInstances(options, callback) {
```
Comment (stephenplusplus) [inline src/instance-group-manager.js]: "getManagedInstances", please!

Comment (modeswitch) [reply src/instance-group-manager.js]: Fixed in f9c0601a7aabd04c29649bfa167e7cd75a48b48a.

**Comment by modeswitch** [comment]
This is ready for kokoro.

**Comment by modeswitch** [comment]
Thanks @JustinBeckwith!

**Comment by modeswitch** [comment]
I think @stephenplusplus needs to approve as well.

**Comment by stephenplusplus** [comment]
Thank you, @modeswitch!

**Pull request merged**